package.json

```diff
@@ -24,7 +24,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node example/index.js",
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha test/**/*.spec.js test/renderers/**/*.spec.js"
```

Comment:
That makes sense. The part I don't understand is `test/**/*.spec.js test/renderers/**/*.spec.js`. Is the second part a subset of the first? Could we simplify to `test/**/*.spec.js`?